File: packages/core-php/src/Cose.php
<?php

declare(strict_types=1);

namespace OpenPasskey;

class Cose
{
    public const ALG_ES256 = -7;
    public const ALG_MLDSA65 = -49;
    public const ALG_COMPOSITE_MLDSA65_ES256 = -52;

    public const KTY_EC2 = 2;
    public const KTY_MLDSA = 8;
    public const KTY_COMPOSITE = 9;

    public const MLDSA_PUB_KEY_SIZE = 1952;
    public const ECDSA_UNCOMPRESSED_SIZE = 65;
}

File: packages/core-ruby/lib/open_passkey/cose.rb
# frozen_string_literal: true

module OpenPasskey
  module Cose
    ALG_ES256 = -7
    ALG_MLDSA65 = -49
    ALG_COMPOSITE_MLDSA65_ES256 = -52

    KTY_EC2 = 2
    KTY_MLDSA = 8
    KTY_COMPOSITE = 9

    MLDSA_PUB_KEY_SIZE = 1952
    ECDSA_UNCOMPRESSED_SIZE = 65
  end
end
